SUSE.rst

```diff
@@ -2,23 +2,22 @@
 Introduction
 ============
 
-Public Cloud Information Service Next Generation (a.k.a. Pint Server NG) enable
-users to lookup SUSE Public Cloud images and services via REST API. Public
-Cloud images and servers data is expected to be managed by the (PostgreSQL)
-database.
+Public Cloud Information Service enable users to lookup SUSE Public Cloud
+images and services via REST API. Public Cloud images and servers data is
+expected to be managed by the (PostgreSQL) database.
```

Comment:
This is generic and not SUSE specific, is already in the README file. SUSE.rst should only contain the SUSE specific information for example that we are deplying it as a lambda function.

Reply:
done